client/lib/wp/sync-handler/cache-index.js

```diff
@@ -20,17 +21,13 @@ import { getLocalForage } from 'lib/localforage';
 const localforage = getLocalForage();
 const debug = debugFactory( 'calypso:sync-handler:cache' );
 
-const RECORDS_LIST_KEY = 'records-list';
-const SYNC_RECORD_REGEX = /^sync-record-\w+$/;
-const LIFETIME = '2 days';
-
 /**
  * Check it the given key is a `sync-record-` key
  *
  * @param {String} key - record key
  * @return {Boolean} `true` if it's a sync-record-<key>
  */
-const isSyncRecordKey = key => SYNC_RECORD_REGEX.test( key );
+const isSyncRecordKey = key => key.indexOf( SYNC_RECORD_NAMESPACE ) === 0;
```

Comment:
hmmm.... I don't think so? It's only used in this module and I don't imagine it being used elsewhere.